labs/lab2/chapter2.md

```diff
@@ -40,21 +40,21 @@ are startup configuration scripts that are executed each time a
 container is started from the image. These scripts configure the
 services and then start them in the running container.
 
-## Building the Docker Image
+## Building the docker image
 
 To build the docker image for this lab please execute the following
 commands:
 
 ```bash
-cd ~/labs/lab2/bigapp/
+cd ~/summit-2017-container-lab/labs/lab2/bigapp/
 docker build -t bigimg .
 ```
 
 This can take a while to build. While you wait you may want to peek at
 the [Review Dockerfile Practices](#review-dockerfile-practices) section 
 at the end of this lab chapter.
 
-## Run Container Based on Docker Image
+## Run Container Based on docker image
```

Comment:
i know there were some of these in the lab1 that you changed as well. My preference would be to lowercase the `d` but keep the other words capitalized. i.e. keep the `I`.